DTS-STN/canadian-dental-care-plan
# Removing Toast. Adding contextual alert for Personal-information.

### Related Azure Boards Work Items
[AB#3308](https://dev.azure.com/DTS-STN/1fc40a8f-28cf-47bc-b6e4-1c234bd06177/_workitems/edit/3308)

### Checklist


### Test Instructions
Navigate to /home and update any personal information. No toast should show up and a ContextualAlert should show up on the persoanl-information page.
